provisioners/roles/newrelic_wsgi/tasks/main.yml

```diff
@@ -9,6 +9,15 @@
     mode=755
     state=directory
   sudo: yes
+
+- name: create directory for newrelic logging accessible to apache user
+  file:
+    dest={{newrelic_agentlog_path}}
+    mode=700
```

Comment:
keep this at 755. the new relic server daemon needs to be able to get into this directory, and 700 locks it down too tight